dark/index.html

```diff
@@ -176,7 +176,7 @@ <h1>
       <div id="tooltip">
         <span id="tooltiptext">GitHub</span>
         <a href="https://github.com/YoshithaRathnayake" target="_blank "><img
-            src="https://img.icons8.com/ios-glyphs/48/github.png" alt="GitHub Icon" /></a>
+            src="https://img.icons8.com/glyph-neue/48/github.png" alt="GitHub Icon" /></a>

```

Comment:
@Rishav1707 Try to add this icon
`https://img.icons8.com/fluency/48/000000/github.png`

Reply:
```suggestion
            src="https://img.icons8.com/fluency/48/000000/github.png" alt="GitHub Icon" /></a>
```